relay-monitors/src/lib.rs

```diff
@@ -113,6 +124,9 @@ struct CheckIn {
     /// Status of this check-in. Defaults to `"unknown"`.
     status: CheckInStatus,
 
+    /// monitor configuration to support upserts.
+    sdk: MinimalClientSdkInfo,
```

Comment:
Old SDKs won't send this, right? So I think this needs to be optional, otherwise deserialization will fail. Make sure to also add a test case to verify this.

```suggestion
    #[serde(default, skip_serializing_if = "Option::is_none")]
    sdk: Option<MinimalClientSdkInfo>,
```

Reply:
Old SDKs don’t support sending monitor checkins however

Reply:
Did we not release an SDK version already with this? If not we can skip this, but if we did, we should make this optional.

Reply:
Chatted w/ @jan-auer I'll go ahead and make this optional